### PR TITLE
Adding docs for new Shifts Connector cmdlets

### PR DESCRIPTION
**Pull request by gucsun**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

File: teams/teams-ps/teams/Get-CsTeamsShiftsConnectionErrorReport.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+This cmdlet returns the list of existing team mapping error reports. It can also return the configuration details for a given mapping result.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-CsTeamsShiftsConnectionErrorReport
+```
+```output
+Code               CreatedAt              Culture Id                                   IntermediateIncident Message                                           Operation                            Parameter   Procedure       ResolvedAt revisitIntervalInMinutes
```
[the remark this answers is not in the document]
Comment (gucsun) [reply teams/teams-ps/teams/Get-CsTeamsShiftsConnectionErrorReport.md:37]: AutoRest removes many plural forms so in the actual response it doesn't have "s".

File: teams/teams-ps/teams/Get-CsTeamsShiftsConnectionInstance.md [hunk 2]
```diff
@@ -34,13 +34,13 @@ This cmdlet returns the list of existing connections. It can also return the con
...
-b75bbfa7-e92b-40fb-99a7-2d23b3404712 {Shift, SwapRequest, OpenShift, OpenShiftRequest…}     {Shift, SwapRequest, OpenShift, TimeOff…}                      "23005736-0000-0400-0000-60d10a990000" WCI-922cb572-369d-42d2-969a-d4927cd3683b Vijay Test Connector PPE Instance DEMO 10                 dfd24b34-ccb0-47e1-bdb…
-cbec0b9a-6161-4f6a-9067-9ec3eb881e56 {Shift, SwapRequest, UserShiftPreferences, OpenShift…} {Shift, SwapRequest, UserShiftPreferences, OpenShift…}         "2300589a-0000-0400-0000-60d132270000" WCI-da788d04-1bd2-48b1-a261-1dd80bde86db Jingjing Connector Instance            10                 dfd24b34-ccb0-47e1-bdb…
+ConnectorAdminEmail     DesignatedActorId                     EnabledConnectorScenario                               EnabledWfiScenario                                             Etag                                   Id                                       Name                                   SyncFrequencyInMin TenantId
```
[the remark this answers is not in the document]
Comment (gucsun) [reply teams/teams-ps/teams/Get-CsTeamsShiftsConnectionInstance.md:37]: Same reason with above

[the remark this answers is not in the document]
Comment (gucsun) [reply teams/teams-ps/teams/Get-CsTeamsShiftsConnectionInstance.md:37]: Same reason with above

[the remark this answers is not in the document]
Comment (gucsun) [reply teams/teams-ps/teams/Get-CsTeamsShiftsConnectionInstance.md:37]: Same reason with above

File: teams/teams-ps/teams/Remove-CsTeamsShiftsScheduleRecord.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+```
+
+## DESCRIPTION
+This cmdlet sends a request of removing Shifts schedule with specified time range.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Remove-CsTeamsShiftsScheduleRecord -TeamId "eddc3b94-21d5-4ef0-a76a-2e4d632e50be" -DateRangeStartDate "2021-09-30T00:00:00" -DateRangeEndDate "2021-10-01T00:00:00" -ClearSchedulingGroup:$false -EntityType "swapRequest", "openShiftRequest" -DesignatedActorId "683af6f2-4f72-4770-b8e1-4ec31836156ad"
```
[the remark this answers is not in the document]
Comment (gucsun) [reply teams/teams-ps/teams/Remove-CsTeamsShiftsScheduleRecord.md:31]: Yes they are. That's why the parameter name is concatenated like "DateRangeStartDate".

File: teams/teams-ps/teams/Remove-CsTeamsShiftsScheduleRecord.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+```
+
+Removes the Shifts schedule record of swapReqeust and openShiftRequest scenarios in the team with ID `eddc3b94-21d5-4ef0-a76a-2e4d632e50be` from 09/30/2021 to 10/01/2021.
```
[the remark this answers is not in the document]
Comment (gucsun) [reply teams/teams-ps/teams/Remove-CsTeamsShiftsScheduleRecord.md:34]: Good catch! Will fix it

File: teams/teams-ps/teams/Get-CsTeamsShiftsConnectionOperation.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+This cmdlet returns the details of a specific batch team mapping operation.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-CsTeamsShiftsConnectionOperation -OperationId c79131b7-9ecb-484b-a8df-2959c7c1e5f2
+```
+```output
+CreatedDateTime      LastActionDateTime   OperationId Status     TenantId                             Type                  WfmConnectorInstanceId
```
[the remark this answers is not in the document]
Comment (gucsun) [reply teams/teams-ps/teams/Get-CsTeamsShiftsConnectionOperation.md:37]: Good catch! Will fix it

[the remark this answers is not in the document]
Comment (gucsun) [reply teams/teams-ps/teams/Get-CsTeamsShiftsConnectionOperation.md:37]: For the brief format of result, some object fields will not show or be blank. You can check the details by adding | format-custom.

File: teams/teams-ps/teams/New-CsTeamsShiftsConnectionBatchTeamMap.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+## SYNTAX
+
+```
+New-CsTeamsShiftsConnectionBatchTeamMap -ConnectorInstanceId <string> -TeamMapping <TeamMap[]> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet connects multiple Microsoft Teams teams and WFM teams to allow for asynchronization of shifts related data. It acts like an async batch request of [New-CsTeamsShiftsConnectionTeamMap](New-CsTeamsShiftsConnectionTeamMap.md). You can check the operation status by running [Get-CsTeamsShiftsConnectionOperation](Get-CsTeamsShiftsConnectionOperation.md).
```
[the remark this answers is not in the document]
Comment (gucsun) [reply teams/teams-ps/teams/New-CsTeamsShiftsConnectionBatchTeamMap.md:28]: No this API is an async one.

File: teams/teams-ps/teams/Get-CsTeamsShiftsConnectionInstance.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (gucsun) [reply teams/teams-ps/teams/Get-CsTeamsShiftsConnectionInstance.md:37]: Some of the object properties are hidden in the brief result, you can get details by adding | format-list or | format-custom

File: teams/teams-ps/teams/Get-CsTeamsShiftsConnectionErrorReport.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gucsun) [reply teams/teams-ps/teams/Get-CsTeamsShiftsConnectionErrorReport.md:41]: I will add other parameters.

**Comment by yogkumgit** [comment]
@dariomws

**Review by dariomws — APPROVED** [review]

**Review by rafaellakeury — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**